configs/scripts/program/dump.sh

```
@@ -29,56 +31,93 @@ if [ ! -d ${OUTPUT} ]; then
-# only prints this if we have external programs
-if [ ${#EXTERNAL_ID[@]} -gt 0 ]; then
-    echo "Dumping external accounts to '${OUTPUT}':"
-fi
-
 # copy external programs or accounts binaries from the chain
 copy_from_chain() {
-    ACCOUNT_TYPE=`echo $1 | cut -d. -f2`
-    PREFIX=$2
+    RPC=$1
+    ACCOUNT_ID=$2
+    ACCOUNT_TYPE=`echo $3 | cut -d. -f2`
```

Shellcheck (0.10.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)